src/util/triplea/MapXMLCreator/TerritoryDefinitionsPanel.java

```diff
@@ -0,0 +1,157 @@
+package util.triplea.MapXMLCreator;
+
+import java.awt.Color;
+import java.awt.Font;
+import java.awt.FontMetrics;
+import java.awt.Graphics;
+import java.awt.Point;
+import java.awt.event.MouseEvent;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Map.Entry;
+
+import javax.swing.JOptionPane;
+import javax.swing.JPanel;
+import javax.swing.SwingUtilities;
+
+import util.triplea.MapXMLCreator.TerritoryDefinitionDialog.DEFINITION;
+
+/**
+ * 
+ * @author Erik von der Osten
+ * 
+ */
+class TerritoryDefinitionsPanel extends ImageScrollPanePanel
+{
+	
+	private TerritoryDefinitionsPanel()
+	{
+		s_instance = this;

```

Comment:
Modifying state of a parent class is brittle, see if this can be avoided. Perhaps have a "TerritoryDefinitionsPanel" static variable instead to take it's place, then split the callers up to call the correct "s_instance"